clients/privacy-center/cypress/e2e/consent-banner.cy.ts

```diff
@@ -1565,7 +1565,7 @@ describe("Consent banner", () => {
         cy.wait("@patchPrivacyPreference").then((preferenceInterception) => {
           const { preferences } = preferenceInterception.request.body;
           const expected = interception.response?.body.map(
-            (s: LastServedNoticeSchema) => s.served_notice_history_id
+            (s: LastServedConsentSchema) => s.served_notice_history_id
```

Comment:
There's a similar `LastServedNoticeSchema` still being imported into clients/privacy-center/cypress/e2e/consent-notices.cy.ts - does that also need to be updated?

Reply:
ahh good eye!! fixed: https://github.com/ethyca/fides/pull/4235/commits/fad6a78fac89628e3c680c933ecf1b66ab97c222